website/docs/docs/permissions/permissions.md

```diff
@@ -40,6 +40,22 @@ Create, update, and delete contract related data for applications that where cre
 
 View any contract related data. This includes applications, application labels, application versions, tags, pacts, and verification results.
 
+## environment:read:*
+
+View a list of all environments.
+
+## environment:manage:*
+
+Create, update, and delete any environment. When creating a new environment the user can associate it with any teams.
+
+## environment:manage:team
```

Comment:
For this one I am wondering if it also makes sense to restrict the delete action?

Reply:
Good point. If you delete an environment, what data is lost?

Reply:
> If you delete an environment, what data is lost?

The record of everything that is deployed to that environment!

The person with :team scope should not be able to delete an environment. The person with * scope should (because someone has to be able to).

I'm actually thinking that maybe we don't even need `environment:manage:team`. The only thing that user can do is modify the display name of the environment, and change its prod status, which is not something that would need to be done often. Maybe we only need `environment:manage:*` and then `environment:read:team` and `environment:read:*` (company picks which one they want to use based on how security conscious they are).

Reply:
It would make it the same as the users, in that currently only someone with user:manage:* can edit users, but the team administrator can add and remove the users from a team.

Reply:
Yeah I agree, It feels like someone with only team level permissions should not be doing much to the environments at all. I'll update to remove the environment:read:team parts